TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/netty/BacksideHttpWatcherHandler.java

```diff
@@ -11,7 +11,7 @@
 @Log4j2
 public class BacksideHttpWatcherHandler extends SimpleChannelInboundHandler<FullHttpResponse> {
 
-    private final AggregatedRawResponse.Builder aggregatedRawResponseBuilder;
+    private AggregatedRawResponse.Builder aggregatedRawResponseBuilder;
     private boolean doneReadingRequest; // later, when connections are reused, switch this to a counter?
     Consumer<AggregatedRawResponse> responseCallback;
 
```

Comment:
Correct, only once & yes, that's why we only set it to once.  And we want to do that so that if a callback is connected AFTER the channelRead is called, we can notice and trigger the callback immediately.